twopane-navigation/src/components/paneRenderer/PaneRenderer.tsx

```diff
@@ -73,12 +73,23 @@ const PaneRenderer = (props: IPaneRendererProps) => {
         
     }
 
-// TODO MOVE TO ONE RENDER
+    const isAccessible = (component: IPaneComponent) => {
```

Comment:
isAccessible() to me implies checking on `accessible` RN prop, I'd rename to avoid confusion -- something like `isVisible` maybe?

Reply:
+1 or 'canNavigateTo' ' canAccess/Open'